packages/offix-client/src/config/ApolloOfflineClientConfigValidator.ts

```diff
@@ -0,0 +1,19 @@
+import { ApolloOfflineClientConfig } from "./ApolloOfflineClientConfig";
+import { ConfigError } from "./ConfigError";
+
+export default function(config: ApolloOfflineClientConfig) {
```

Comment:
nit: we tend to avoid default exports in Offix. Additionally, anonymous functions (functions with no name) are not great because if an error is thrown from them the stack trace will say something like `anonymous` function instead of the name of the function.

Could we change it to something like `export function validateConfig(config: ApolloOfflineClientConfig)`?

Reply:
I'll change it